chain/network/src/peer_manager/peer_manager_actor.rs

```diff
@@ -1394,33 +1382,19 @@ impl PeerManagerActor {
     /// Send message to peer that belong to our active set
     /// Return whether the message is sent or not.
     fn send_message(
-        &mut self,
-        ctx: &mut Context<Self>,
+        connected_peers: &HashMap<PeerId, ConnectedPeer>,
         peer_id: PeerId,
         message: PeerMessage,
     ) -> bool {
-        if let Some(active_peer) = self.connected_peers.get(&peer_id) {
+        if let Some(connected_peer) = connected_peers.get(&peer_id) {
             let msg_kind = message.msg_variant().to_string();
             trace!(target: "network", ?msg_kind, "Send message");
-            active_peer
-                .addr
-                .send(SendMessage { message })
-                .into_actor(self)
-                .map(move |res, act, _| {
-                    res.map_err(|e| {
-                        // Peer could have disconnect between check and sending the message.
-                        if act.connected_peers.contains_key(&peer_id) {
-                            error!(target: "network", ?msg_kind, ?e, "Failed sending message")
-                        }
-                    })
-                })
-                .map(|_, _, _| ())
-                .spawn(ctx);
+            actix::spawn(connected_peer.addr.send(SendMessage { message }));
             true
         } else {
             debug!(target: "network",
                    to = ?peer_id,
-                   num_connected_peers = self.connected_peers.len(),
+                   num_connected_peers = connected_peers.len(),
```

Comment:
hmm.. why does this debug log says 'sending message' - if message is not really sent?

Reply:
I changed it to "failed sending message"